GTHC/web
# Countdown Widget and Twitter Widgets

## Description

The following PR involves adding more widgets to the Home Page + fixing its layout. This adds both twitter widgets (DukeMMB and KvilleNation) to the home page and also adds a dynamic countdown timer for the Duke v.s UNC game. It also switches the overall layout of the homepage from a container to a grid for better organization/widget layout. 

## Motivation and Context


Need a nice UI + helpful widgets on the homepage to increase the likelihood someone would use this application. 
## How Has This Been Tested?



Relatively static content, so testing hasn't been substantial yet (Will be tested more once we as a group start the testing phase). Testing was done through the console to make sure there weren't any errors or resizing issues. The accuracy of the countdown widget was compared to an online timer to make sure the date being shown was correct. 
1. Yarn dev:start
2. Login/Sign Up 
3. Go to Home Page on Top

## Screenshots (if appropriate):

<img width="1440" alt="screen shot 2019-01-04 at 2 58 14 am" src="https://user-images.githubusercontent.com/22209648/50678873-cc7e0000-0fce-11e9-94bb-029e8b54caff.png">

## Types of changes


## Checklist:
